support/elasticsearch2_search.erl

```diff
@@ -374,7 +388,7 @@ map_query({text, <<"id:", _/binary>>}, _Context) ->
     false;
 map_query({text, Text}, Context) ->
     DefaultFields = [
-        <<"_all">>,
+        <<"*">>,
```

Comment:
This only works in newer versions of Elastic, but this module is also supposed to work with Elastic 5 right?

Reply:
Indeed, this module was made to support 6+